### PR TITLE
Rename POSIX-API, move Configuration, User-CLI and Web-Interface

### PR DESCRIPTION
**Pull request by LuQQiu**
For Chinese docs
Rename FUSE-API.md -> POSIX-API.md
Move cn/basic/Command-Line-Interface.md to /cn/operation/User-CLI.md
Move cn/basic/Configuration-Settings.md to cn/operation/Configuration.md
Move cn/basic/Web-Interface.md to cn/operation/Web-Interface.md
and change related urls

### CODE REVIEW

**Comment by LuQQiu** [comment]
@Xenorith PTAL the URL changes and rename changes
@jiacheliu3  PTAL the chinese changes
Thanks!

File: docs/cn/ufs/COS.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ fs.cos.app.id=<COS_APP_ID>
 
 ### 嵌套目录安装
 
-COS可以安装在Alluxio命名空间中的嵌套目录中，以统一访问多个存储系统。 [Mount命令]({{ '/cn/basic/Command-Line-Interface.html' | relativize_url }}#mount)可以实现这一目的。例如，下面的命令将COS实例内部的目录挂载到Alluxio的/cos目录
+COS可以安装在Alluxio命名空间中的嵌套目录中，以统一访问多个存储系统。 [Mount命令](({{ '/cn/operation/User-CLI.html' | relativize_url }}#mount)可以实现这一目的。例如，下面的命令将COS实例内部的目录挂载到Alluxio的/cos目录
```
Comment (Xenorith) [inline docs/cn/ufs/COS.md]: extra `(`

File: docs/_data/table/cn/Developer-Tips.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ formatWorker:
 bootstrapConf:
   如果引导程序配置文件<code>alluxio-env.sh</code>不存在，利用<code>ALLUXIO_MASTER_HOSTNAME</code>生成。
 fs:
-  在命令行中与Alluxio进行交互，从而完成基本的文件系统操作。详情见<a href="Command-Line-Interface.html">Command Line</a>。
+  在命令行中与Alluxio进行交互，从而完成基本的文件系统操作。详情见[用户命令行](({{ '/cn/operation/User-CLI.html' | relativize_url }})。
```
Comment (Xenorith) [inline docs/_data/table/cn/Developer-Tips.yml]: extra `(`

File: docs/cn/advanced/Security.md [hunk 3]
```diff
@@ -100,9 +100,9 @@ drwxr-xr-x jack           staff                       24       PERSISTED 11-20-2
 
 1. 用户应用可以调用`FileSystem API`或`Hadoop API`的`setAttribute(...)`方法，参考[文件系统API](File-System-API.html)。
 2. CLI命令，参考
-[chown](Command-Line-Interface.html#chown)，
-[chgrp](Command-Line-Interface.html#chgrp)，
-[chmod](Command-Line-Interface.html#chmod)。
+[chown](({{ '/cn/operation/User-CLI.html' | relativize_url }}#chown)，
```
Comment (Xenorith) [inline docs/cn/advanced/Security.md]: extra `(`s for all 3

File: docs/cn/advanced/Time-To-Live.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ SetTTL(path, duration, action)
 
 ### 使用命令行
 
-查看[命令行文档](Command-Line-Interface.html#setttl).
+查看[命令行文档](({{ '/cn/operation/User-CLI.html' | relativize_url }}#setttl).
```
Comment (Xenorith) [inline docs/cn/advanced/Time-To-Live.md]: extra `(`

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 5]
```diff
@@ -226,7 +226,7 @@ $ git push origin awesome_feature
 
 1. [本地运行Alluxio](Running-Alluxio-Locally.html)
 1. [集群上运行Alluxio](Running-Alluxio-on-a-Cluster.html)
-1. 阅读[配置设置](Configuration-Settings.html)和[命令行相关操作](Command-Line-Interface.html)
+1. 阅读[配置设置]({{ '/cn/operation/Configuration.html' | relativize_url }})和[命令行相关操作](({{ '/cn/operation/User-CLI.html' | relativize_url }})
```
Comment (Xenorith) [inline docs/cn/contributor/Contributor-Getting-Started.md]: extra `(` in second url

File: docs/_config.yml [hunk 6]
```diff
@@ -60,13 +60,13 @@ categoryInfo:
     cn: 用户指南
   Core Services:
     en: Core Services
-    cn: TODO
+    cn: 核心功能
```
Comment (Xenorith) [inline docs/_config.yml]: google translate says this is "core functions" :D

File: docs/_config.yml [hunk 7]
```diff
@@ -60,13 +60,13 @@ categoryInfo:
...
   Install Alluxio:
     en: Install Alluxio
     cn: 部署 Alluxio
   Alluxio in the Cloud:
     en: Alluxio in the Cloud
-    cn: TODO
+    cn: Alluxio云端部署与应用
```
Comment (Xenorith) [inline docs/_config.yml]: and this is "Alluxio cloud deployment and application" wow it is good

File: docs/_config.yml (hunk 6 above)
Comment (LuQQiu) [reply docs/_config.yml]: Haha, yeah, it's kind of core functions if translate directly. But `function` in Chinese means what an application used for. The direct translation of `services` in Chinese sounds strange. Leave it to @jiacheliu3!!

File: docs/_config.yml (hunk 7 above)
Comment (LuQQiu) [reply docs/_config.yml]: Yeah that's exactly what it means!

File: docs/cn/advanced/Alluxio-Storage-Management.md [hunk 8]
```diff
@@ -115,7 +115,7 @@ Alluxio支持自定义回收策略，已有的实现包括：
 
 ### 开启和配置分层存储
 
-在Alluxio中，使用[配置参数](Configuration-Settings.html)开启分层存储。使用如下配置参数可以指定Alluxio的额外存储层：
+在Alluxio中，使用[配置参数]({{ '/cn/operation/Configuration.html' | relativize_url }})开启分层存储。使用如下配置参数可以指定Alluxio的额外存储层：
```
Comment (jiacheliu3) [inline docs/cn/advanced/Alluxio-Storage-Management.md]: I don't think Configuration.html page has anything with Tiered Store. Maybe you should link to https://docs.alluxio.io/ee/user/stable/en/advanced/Alluxio-Storage-Management.html

Comment (jiacheliu3) [reply docs/cn/advanced/Alluxio-Storage-Management.md]: Or if this should be the list of config properties should it be https://docs.alluxio.io/ee/user/stable/en/reference/Properties-List.html?

File: docs/cn/advanced/Security.md [hunk 9]
```diff
@@ -21,7 +21,7 @@ Alluxio还支持其它身份验证模式，如`NOSASL`和`CUSTOM`。
 1. [审计](#auditing): 如果是 `alluxio.master.audit.logging.enabled=true`， Alluxio 文件系统
 维护用户访问文件元数据的审计日志(audit log)。
 
-参考[安全性配置项](Configuration-Settings.html#security-configuration)的信息以启用不同安全特性。
+参考[安全性配置项]({{ '/cn/operation/Configuration.html' | relativize_url }}#security-configuration)的信息以启用不同安全特性。
```
Comment (jiacheliu3) [inline docs/cn/advanced/Security.md]: Doesn't seem to me the original `Configuration-Settings.html#security-configuration` section exists anymore. Should this be changed to sth else?

File: docs/cn/compute/Spark.md [hunk 10]
```diff
@@ -133,7 +133,7 @@ $ spark-submit \
 ...
 ```
 
-如果需要自定义 Spark 作业中的 Alluxio 客户端侧属性，请参见[如何配置 Spark 作业]({{ '/cn/basic/Configuration-Settings.html' | relativize_url }}#spark)。
+如果需要自定义 Spark 作业中的 Alluxio 客户端侧属性，请参见[如何配置 Spark 作业]({{ '/cn/operation/Configuration.html' | relativize_url }}#spark)。
```
Comment (jiacheliu3) [inline docs/cn/compute/Spark.md]: Spark作业是啥？？

File: docs/_config.yml (hunk 6 above)
Comment (jiacheliu3) [reply docs/_config.yml]: I'd say Lu's translation makes more sense.

File: docs/_config.yml [hunk 11]
```diff
@@ -60,13 +60,13 @@ categoryInfo:
     cn: 用户指南
```
Comment (Xenorith) [inline docs/_config.yml]: oh can you also remove the `# TODO:` line above this?

File: docs/cn/advanced/Alluxio-Storage-Management.md (hunk 8 above)
Comment (LuQQiu) [reply docs/cn/advanced/Alluxio-Storage-Management.md]: Looks at the original en docs, the url is correct, but his way of translation makes the url confusing

File: docs/cn/advanced/Security.md (hunk 9 above)
Comment (LuQQiu) [reply docs/cn/advanced/Security.md]: This should be properties-list docs

File: docs/cn/compute/Spark.md (hunk 10 above)
Comment (LuQQiu) [reply docs/cn/compute/Spark.md]: change to 任务

File: docs/_config.yml (hunk 11 above)
Comment (LuQQiu) [reply docs/_config.yml]: Removed

File: docs/cn/advanced/Alluxio-Storage-Management.md [hunk 12]
```diff
@@ -21,7 +21,7 @@ Alluxio管理了Alluxio workers节点的包括内存在内的本地存储，来
 
 默认配置下的Alluxio将为每个worker提供一个ramdisk，并占用一定的百分比的系统总内存。这个ramdisk将被用作分配给每个Alluxio worker的唯一存储介质。
 
-Alluxio存储通过Alluxio的`alluxio-site.properties`配置。详细配置请参考[配置文档](Configuration-Settings.html)。
+Alluxio存储通过Alluxio的`alluxio-site.properties`配置。详细配置请参考[配置文档]({{ '/cn/operation/Configuration.html' | relativize_url }})。
```
Comment (jiacheliu3) [inline docs/cn/advanced/Alluxio-Storage-Management.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/advanced/Security.md [hunk 13]
```diff
@@ -186,7 +186,7 @@ Alluxio审计日志格式如下表所示：
 
 它和HDFS审计日志的格式[wiki](https://wiki.apache.org/hadoop/HowToConfigure)很像。
 
-为了使用Alluxio的审计功能，你需要将JVM参数`alluxio.master.audit.logging.enabled`设置为true，具体可见[Configuration settings](Configuration-Settings.html)。
+为了使用Alluxio的审计功能，你需要将JVM参数`alluxio.master.audit.logging.enabled`设置为true，具体可见[Configuration settings]({{ '/cn/operation/Configuration.html' | relativize_url }})。
```
Comment (jiacheliu3) [inline docs/cn/advanced/Security.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/advanced/Tiered-Locality.md [hunk 14]
```diff
@@ -22,7 +22,7 @@ priority: 1
 alluxio.locality.[tiername]=...
 ```
 
-请参阅[配置项设置](Configuration-Settings.html)页面以获取详细信息来设置配置属性。
+请参阅[配置项设置]({{ '/cn/operation/Configuration.html' | relativize_url }})页面以获取详细信息来设置配置属性。
```
Comment (jiacheliu3) [inline docs/cn/advanced/Tiered-Locality.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/advanced/Time-To-Live.md [hunk 15]
```diff
@@ -26,7 +26,7 @@ TTL间隔延迟。间隔长度由 `alluxio.master.ttl.checker.interval` 属性
 alluxio.master.ttl.checker.interval=10m
 ```
 
-查看[配置页](Configuration-Settings.html)获取有关Alluxio设置的更多配置
+查看[配置页]({{ '/cn/operation/Configuration.html' | relativize_url }})获取有关Alluxio设置的更多配置
```
Comment (jiacheliu3) [inline docs/cn/advanced/Time-To-Live.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/api/Clients-Java.md [hunk 16]
```diff
@@ -85,7 +85,7 @@ Alluxio提供了位置策略来选择要存储文件块到哪一个worker。
 
 使用Alluxio的Java API，用户可以设置策略在`CreateFileOptions`中向Alluxio写入文件和在`OpenFileOptions`中读取文件。
 
-用户可以轻松地覆盖默认的策略类[配置文件](Configuration-Settings.html)中的属性`alluxio.user.block.write.location.policy.class`。内置的策略包括：
+用户可以轻松地覆盖默认的策略类[配置文件]({{ '/cn/operation/Configuration.html' | relativize_url }})中的属性`alluxio.user.block.write.location.policy.class`。内置的策略包括：
```
Comment (jiacheliu3) [inline docs/cn/api/Clients-Java.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/compute/Presto.md [hunk 17]
```diff
@@ -117,7 +117,7 @@ Presto 服务器日志：
 
 ### 自定义 Alluxio 用户属性
 
-要配置其他 Alluxio 属性，可以将包含[`alluxio-site.properties`]({{ '/en/basic/Configuration-Settings.html' | relativize_url }})的配置路径（即`${ALLUXIO_HOME}/conf`）追加到 Presto 文件夹下的`etc/jvm.config`的 JVM 配置中。
+要配置其他 Alluxio 属性，可以将包含[`alluxio-site.properties`]({{ '/cn/operation/Configuration.html' | relativize_url }})的配置路径（即`${ALLUXIO_HOME}/conf`）追加到 Presto 文件夹下的`etc/jvm.config`的 JVM 配置中。
```
Comment (jiacheliu3) [inline docs/cn/compute/Presto.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/api/FS-API.md [hunk 18]
```diff
@@ -112,7 +112,7 @@ Alluxio支持自定义策略，所以你可以通过实现接口`alluxio.client.
 
 Alluxio允许客户端在向本地worker写入数据块时选择偏好的存储层。目前这种策略偏好只存在于本地worker，不支持远程workers; 远程worker会将数据块写到最高存储层。
 
-默认情况下，数据写入顶层。 用户可以通过修改`alluxio.user.file.write.tier.default` [配置文件](Configuration-Settings.html)属性改变默认设置，或通过`FileSystem#createFile(AlluxioURI)`的API调用选项覆盖默认设置。
+默认情况下，数据写入顶层。 用户可以通过修改`alluxio.user.file.write.tier.default` [配置文件]({{ '/cn/operation/Configuration.html' | relativize_url }})属性改变默认设置，或通过`FileSystem#createFile(AlluxioURI)`的API调用选项覆盖默认设置。
```
Comment (jiacheliu3) [inline docs/cn/api/FS-API.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/api/FS-API.md [hunk 19]
```diff
@@ -84,7 +84,7 @@ Alluxio提供定位策略，用于确定应该选择哪个Worker来存储文件
 
 使用Alluxio的Java API，用户可以在`CreateFileOptions`中设置该策略以用于写文件，也可在`OpenFileOptions`中设置该策略用于向Alluxio中读文件。
 
-用户可以简单的覆盖默认策略类通过修改[配置文件](Configuration-Settings.html)`alluxio.user.block.write.location.policy.class`内的属性。内置策略包括：
+用户可以简单的覆盖默认策略类通过修改[配置文件]({{ '/cn/operation/Configuration.html' | relativize_url }})`alluxio.user.block.write.location.policy.class`内的属性。内置策略包括：
```
Comment (jiacheliu3) [inline docs/cn/api/FS-API.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 20]
```diff
@@ -226,7 +226,7 @@ $ git push origin awesome_feature
 
 1. [本地运行Alluxio](Running-Alluxio-Locally.html)
 1. [集群上运行Alluxio](Running-Alluxio-on-a-Cluster.html)
-1. 阅读[配置设置](Configuration-Settings.html)和[命令行相关操作](Command-Line-Interface.html)
+1. 阅读[配置设置]({{ '/cn/operation/Configuration.html' | relativize_url }})和[命令行相关操作]({{ '/cn/operation/User-CLI.html' | relativize_url }})
```
Comment (jiacheliu3) [inline docs/cn/contributor/Contributor-Getting-Started.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/deploy/Running-Alluxio-Yarn-Integration.md [hunk 21]
```diff
@@ -18,7 +18,7 @@ priority: 4
 **下载到本地的 [Alluxio](https://alluxio.io/download)**
 
 ## 配置
-通过一些特定的属性来定制Alluxio master和worker(比如，在各个worker节点上设置分层存储)，查阅[Configuration settings](Configuration-Settings.html)
+通过一些特定的属性来定制Alluxio master和worker(比如，在各个worker节点上设置分层存储)，查阅[Configuration settings]({{ '/cn/operation/Configuration.html' | relativize_url }})
```
Comment (jiacheliu3) [inline docs/cn/deploy/Running-Alluxio-Yarn-Integration.md]: Change to `/en/reference/Properties-List.html`?

Comment (jiacheliu3) [reply docs/cn/deploy/Running-Alluxio-Yarn-Integration.md]: Configuration settings改成中文？

File: docs/cn/deploy/Running-Alluxio-on-Mesos.md [hunk 22]
```diff
@@ -49,7 +49,7 @@ alluxio.worker.web.port=31500
 
 #### 配置属性
 
-参考[配置项设置](Configuration-Settings.html)文档获取如何配置相应属性。
+参考[配置项设置]({{ '/cn/operation/Configuration.html' | relativize_url }})文档获取如何配置相应属性。
```
Comment (jiacheliu3) [inline docs/cn/deploy/Running-Alluxio-on-Mesos.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/operation/Configuration.md [hunk 23]
```diff
@@ -83,7 +83,7 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 如果首选的是数据持久性的部署，所有的作业都需要写到UFS和Alluxio，那么就可以使用Alluxio v1.8或更高版本的admin命令来简单地添加`alluxio.user.file.writetype.default=CACHE_THROUGH` 到master端`${ALLUXIO_HOME}/conf/alluxio-site.properties`。重新启动集群后，所有新的作业都将自动将属性`alluxio.user.file.writetype.default`设置为`CACHE_THROUGH` 
 
 客户端仍然可以忽略或覆盖集群范围内的默认值，通过指定属性`alluxio.user.conf.cluster.default.enabled=false`，
-以更改加载集群范围内的默认值，或者遵循前面描述的方法[为应用程序配置文件](Configuration-Settings.html#configure-applications)覆盖相同的属性。
+以更改加载集群范围内的默认值，或者遵循前面描述的方法[为应用程序配置文件]({{ '/cn/operation/Configuration.html' | relativize_url }}#configure-applications)覆盖相同的属性。
```
Comment (jiacheliu3) [inline docs/cn/operation/Configuration.md]: Change to `/en/reference/Properties-List.html`?

File: docs/cn/operation/Troubleshooting.md [hunk 24]
```diff
@@ -125,7 +125,7 @@ $ ls {{site.ALLUXIO_CLIENT_JAR_PATH}}
 - 请确保Alluxio的master节点和client节点的安全设置保持一致.
 Alluxio通过配置`alluxio.security.authentication.type`来提供不同的用户身份验证(Security.html#authentication)的方法。
 如果客户端和服务器的这项配置属性不一致，这种错误将会发生。(例如，客户端的属性为默认值`NOSASL`,而服务器端设为`SIMPLE`)
-有关如何设定Alluxio的集群和应用的问题，用户请参照[Configuration-Settings](Configuration-Settings.html)
+有关如何设定Alluxio的集群和应用的问题，用户请参照[Configuration-Settings]({{ '/cn/operation/Configuration.html' | relativize_url }})
```
Comment (jiacheliu3) [inline docs/cn/operation/Troubleshooting.md]: Change to `/en/reference/Properties-List.html`?

Configuration-Settings改成中文？

File: docs/cn/advanced/Alluxio-Storage-Management.md (hunk 12 above)
Comment (LuQQiu) [reply docs/cn/advanced/Alluxio-Storage-Management.md]: this one is Configuration.html in en docs
讲如何配置是configuration.html

File: docs/cn/advanced/Security.md (hunk 13 above)
Comment (LuQQiu) [reply docs/cn/advanced/Security.md]: This one is also configuration.html

File: docs/cn/advanced/Tiered-Locality.md (hunk 14 above)
Comment (LuQQiu) [reply docs/cn/advanced/Tiered-Locality.md]: Same, configuration

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**